EasyPost/Address.cs

```diff
@@ -92,8 +87,8 @@ public static async Task<AddressCollection> All(Dictionary<string, object>? para
         ///     * {"country", string}
         ///     * {"phone", string}
         ///     * {"email", string}
-        ///     * {"verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
-        ///     * {"strict_verifications", List&lt;string&gt;} Possible items are "delivery" and "zip4".
+        ///     * {"verifications", List&lt;Any&gt;} Possible items are true, "delivery" and "zip4".
+        ///     * {"strict_verifications", List&lt;Any&gt;} Possible items are true, "delivery" and "zip4".
```

Comment:
@nwithan8 can you help me update this docstring to say either a bool, string, or list of any?